packages/wallets/src/features/cashier/modules/WithdrawalCrypto/utils/withdrawalCryptoValidators.ts

```diff
@@ -69,12 +67,14 @@ const validateCryptoInput = (
 
     const amount = parseFloat(value);
 
+    if (amount > activeWallet.balance) return helperMessageMapper.insufficientFunds;
+
     const MIN_WITHDRAWAL_AMOUNT = activeWallet.currency_config.minimum_withdrawal;
 
-    const MAX_WITHDRAWAL_AMOUNT = remainder < activeWallet.balance ? remainder : activeWallet.balance;
+    const MAX_WITHDRAWAL_AMOUNT =
+        !isClientVerified && remainder < activeWallet.balance ? remainder : activeWallet.balance;
 
-    if (isClientVerified && amount > activeWallet.balance) return helperMessageMapper.insufficientFunds;
-    else if (MIN_WITHDRAWAL_AMOUNT && (amount < MIN_WITHDRAWAL_AMOUNT || amount > MAX_WITHDRAWAL_AMOUNT)) {
+    if (MIN_WITHDRAWAL_AMOUNT && (amount < MIN_WITHDRAWAL_AMOUNT || amount > MAX_WITHDRAWAL_AMOUNT)) {
```

Comment:
just to confirm - is it okay that its not gonna validate MAX_WITHDRAWAL_AMOUNT when MIN_WITHDRAWAL_AMOUNT is not truthy?

Reply:
actually, I see that it was there before, so at least its not regression,

Reply:
Yes. It won't if MIN_WITHDRAWAL is `undefined`.